app/networkapi/people/views.py

```diff
@@ -1,3 +1,28 @@
 from django.shortcuts import render
+from rest_framework.generics import ListAPIView, RetrieveAPIView
+from networkapi.people.models import Person
+from networkapi.people.serializers import PersonSerializer
 
-# Create your views here.
+
+class PeopleListView(ListAPIView):
+    """
+    A view that permits a GET to allow listing all the People
+    in the database
+
+    **Route** - `/people`
```

Comment:
I think let's remove the "route" part since typically, views aren't tied to urls.